app/api/events.py

```diff
@@ -210,15 +278,33 @@ def before_get_object(self, view_kwargs):
                 else:
                     view_kwargs['id'] = None
 
-    decorators = (is_coorganizer, )
+    def after_get(self, result):
+        result = changeToTimezone(result)
```

Comment:
@niranjan94
Checking this .
So what we need is  2017-07-12 05:52:33+05:30 in the database and same would be the output and then `changeToTimezone` function won't be needed. 
Correct me if I'm wrong.
Working on this.